paasta_tools/kubernetes_tools.py

```diff
@@ -921,7 +933,7 @@ def get_autoscaling_metric_spec(
         hpa = V2beta2HorizontalPodAutoscaler(
             kind="HorizontalPodAutoscaler",
             metadata=V1ObjectMeta(
-                name=name, namespace=namespace, annotations=annotations, labels=labels
+                name=name, namespace=namespace, annotations=dict(), labels=labels
```

Comment:
comment: heh, i was confused by this diff until i read the previous code and realized (as you did) that we never wrote to `annotations` 🤣